user/stubs.c

```diff
@@ -0,0 +1,119 @@
+#include "userargs.h"
+#include <sys/syscall.h>
+
+sgxlkl_userargs_t* __sgxlkl_userargs;
+
+void sgxlkl_warn(const char* fmt, ...);
+
+int snprintf(char *str, size_t size, const char *format, ...);
+
+/*
+**==============================================================================
+**
+** syscall:
+**
+**==============================================================================
+*/
+
+long lkl_syscall(long no, long* params)
+{
+    long ret = __sgxlkl_userargs->ua_lkl_syscall(no, params);
+
+    return ret;
+}
+
+long __sgxlkl_log_syscall(
+    // sgxlkl_syscall_kind type,
+    uint32_t type,
+    long n,
+    long res,
+    int params_len,
+    ...)
+{
+    sgxlkl_warn("__sgxlkl_log_syscall() unimplemented in user space");
```

Comment:
This is still called for `dup2` and `epoll_wait` (redirected system calls).  I think we should just remove the caller here, since we're removing the behaviour.